src/Control/Parallel.purs

```diff
@@ -48,3 +50,13 @@ parSequence_
   => t (m a)
   -> m Unit
 parSequence_ = parTraverse_ id
+
+parOneOf
+  :: forall a t m f
+   . Parallel f m
+  => Plus f
```

Comment:
I was just providing the minimal set of constraints required to get this to compile.